src/components/Genres/Genres.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import styles from "./Genres.module.scss";
+
+const Genres: React.FC<{
+  genres: string[];
+  selectedGenre: string;
+  onSelect: (genre: string) => void;
+}> = ({ genres, selectedGenre, onSelect }) => {
```

Comment:
It is common practice to define props separately with either type or interface, and then just specify it here (e.g. `React.FC<GenresProps>`). It can be defined on the top of the file or in a separate file. 
In fact, using `React.FC` is not recommended these days, you could as well do without it, like so:

`const Genres = ({ genres, selectedGenre, onSelect }: GenresProps) => { ... }`